submissions/drewzag/hooli-style-popup/index.html

```diff
@@ -0,0 +1,173 @@
+                    <span class="popup-text">News</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/maps.png" alt="" class="popup-image" />
+                    <span class="popup-text">Maps</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/files.png" alt="" class="popup-image" />
+                    <span class="popup-text">Files</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/calendar.png" alt="" class="popup-image" />
+                    <span class="popup-text">Contacts</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/translation.png" alt="" class="popup-image" />
+                    <span class="popup-text">Translate</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/videos.png" alt="" class="popup-image" />
+                    <span class="popup-text">Video</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/search.png" alt="" class="popup-image" />
+                    <span class="popup-text">Search</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/photos.png" alt="" class="popup-image" />
+                    <span class="popup-text">Photo</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/apps.png" alt="" class="popup-image" />
+                    <span class="popup-text">Apps</span>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__item-link">
+                    <img src="/popup-icons/documents.png" alt="" class="popup-image" />
+                    <span class="popup-text">Document</span>
+                  </a>
+                </li>
+              </ul>
+
+              <input type="checkbox" class="more__checkbox" id="checkbox__more-less" />
+              <label for="checkbox__more-less">
+                <span class="show-more">Show More</span>
+              </label>
```

Comment:
Please fix styles on the focus state to indicate the current position of focus on the page. Watch provided  [video](https://share.vidyard.com/watch/zy6cyn6ShCLu43EeFyo2Qc?)